src/main/scala/mesosphere/mesos/Constraints.scala

```diff
@@ -86,77 +90,51 @@ object Constraints {
       groupedTasks.find(_._1.contains(constraintValue)).forall(_._2 < maxCount)
     }
 
-    private def checkHostName =
-      constraint.getOperator match {
-        case Operator.LIKE => offer.getHostname.matches(value)
-        case Operator.UNLIKE => !offer.getHostname.matches(value)
-        // All running tasks must have a hostname that is different from the one in the offer
-        case Operator.UNIQUE => allPlaced.forall(_.hostname != offer.getHostname)
-        case Operator.GROUP_BY => checkGroupBy(offer.getHostname, (p: Placed) => Some(p.hostname))
-        case Operator.MAX_PER => checkMaxPer(offer.getHostname, value.toInt, (p: Placed) => Some(p.hostname))
-        case Operator.CLUSTER =>
-          // Hostname must match or be empty
-          (value.isEmpty || value == offer.getHostname) &&
-            // All running tasks must have the same hostname as the one in the offer
-            allPlaced.forall(_.hostname == offer.getHostname)
-        case _ => false
-      }
+    private def checkCluster(ov: String, placedValue: Placed => Option[String]) = {
```

Comment:
Do you think we can use another name instead of `ov`?

Reply:
✅